test/error-handler.test.js

```diff
@@ -9,59 +9,43 @@ var assert = require('assert');
 var request = require('supertest');
 
 describe('loopback.errorHandler(options)', function() {
-  it('should return default middleware when options object is not present', function(done) {
-    //arrange
-    var app = loopback();
-    app.use(loopback.urlNotFound());
-    app.use(loopback.errorHandler({ log: false }));
-
-    //act/assert
-    request(app)
-      .get('/url-does-not-exist')
-      .end(function(err, res) {
-        assert.ok(res.error.text.match(/<ul id="stacktrace"><li> &nbsp; &nbsp;at raiseUrlNotFoundError/));
-
-        done();
-      });
+  it('should pass strong-error-handler message to migrate', function(done) {
+    try {
+  //arrange
+      var app = loopback();
+      app.use(loopback.urlNotFound());
+      app.use(loopback.errorHandler({ log: false }));
```

Comment:
I hope you meant to use that inside the `catch`, since the test case fails without try/catch. 
It is inside the catch and working fine.